R/apply_fmt_fn.R

```diff
@@ -71,7 +71,7 @@ alias_as_fmt_fn <- function(x, call = parent.frame()) {
     return(label_cards(digits = as.integer(x)))
   }
   if (is_string(x)) {
-    .check_fmt_string(x, call = call)
+    .check_fmt_string(x, call = call) # .check_fmt_string() does have different messaging than the cli_abort() below was that intended?
```

Comment:
Because `apply_fmt_fn()` is a wrapper for `alias_as_fmt_fn()` which contains `.check_fmt_string()` was the intention to have two separate error messages intentional?